### DISCUSSION TITLE
希望在附件上传功能中可以增加挂载到Webdav功能

### DISCUSSION THREAD

**Discussion by Austinwenyz** [halo-dev/halo#4794] [Ideas]
### 你当前使用的版本

2.0.1

### 描述一下此特性

因为部署在服务器上的储存空间较小，因此希望可以把文件储存在webdav上。

### 附加信息

尝试过使用rclone直接挂载附件上传文件夹，但是不起作用，因此希望可以直接增加挂载点到webdav，就像附件上传到阿里OSS到插件一样

十分感谢！

**Comment by ruibaby** [comment]
请参考：https://github.com/halo-dev/halo/issues/1672

/kind feature
/area plugin
